framework/metricsresource/error.go

```diff
@@ -0,0 +1,12 @@
+package metricsresource
+
+import (
+	"github.com/juju/errgo"
```

Comment:
Looks like `errgo` is still used across the `operatorkit` library. I can migrate next in a separate PR.